code/modules/mob/living/carbon/human/species_types/monkeys.dm

```diff
@@ -37,7 +37,6 @@
 	dust_anim = "dust-m"
 	gib_anim = "gibbed-m"
 
-	payday_modifier = 1.5
```

Comment:
If this PR is being updated to simply change the value, make sure you also put this back.

Reply:
Has its own define.